ex6_2_3.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+
+def timer(func,*args,**kwargs)->time:
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names